edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -376,6 +380,48 @@ Option<T> GetConfigIfExists<T>(string fieldName, IConfiguration configuration, I
             }
 
             return EqualityComparer<T>.Default.Equals(storageParamValue, default(T)) ? Option.None<T>() : Option.Some(storageParamValue);
+        }
+
+        void ValidateStorageIdentity(string storagePath)

```

Comment:
Please refactor this to the Util project. Please work with @ancaantochi if you need help..

Reply:
Have these changes done, just not pushed. I was waiting on some input to see if it was recommended. Will address the other comments there and push that instead.